.travis/push-image.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+if [[ "$TRAVIS_BRANCH" = "master" && "$TRAVIS_PULL_REQUEST" = "false" ]]; then
+    echo "$DOCKER_PASSWORD" | docker login --username "$DOCKER_USERNAME" --password-stdin 
+
+    # tag temporarily as liberoadmin due to lack of `libero/` availability
+    docker tag "libero/browser:$IMAGE_TAG" "liberoadmin/browser:$IMAGE_TAG"
+    docker push "liberoadmin/browser:$IMAGE_TAG"
+else
+    echo "not pushing an image, we are not on master"
```

Comment:
Fine for now. We can improve messages when we update to use the proper organisation and handle tags.